bitcoin/electrs/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM rust:alpine3.13
+
+RUN apk add clang-dev build-base linux-headers
+RUN rustup component add rustfmt
+WORKDIR /workdir
```

Comment:
Is this line needed here? We're doing COPY next with an absolute destination path and after that, we're setting another WORKDIR.

Reply:
good catch :D